### PR TITLE
Moving info from readme to Documentation folder as per issue 14

### PR DESCRIPTION
**Pull request by LiamBailey**
I have moved some sections from the readme into their own file in a new folder called documentation. Please let me know if this is what you wanted, and if you want any of the other sections bringing into documentation also.

### CODE REVIEW

**Comment by pxgamer** [comment]
Hi, apologies for my late reply.

I'll review this today, thank you!

File: README.md [hunk 1]
```diff
@@ -11,152 +11,19 @@ An API wrapper for the Arionum cryptocurrency node.
 
 ## Install
 
-Via Composer
-
-```bash
-$ composer require pxgamer/arionum-php
-```
+Please see [Installation Doc](documentation/installation.md) for full information on how to use this library.
```
[the remark this answers is not in the document]
Comment (LiamBailey) [reply README.md]: OK I will change this now

**Comment by LiamBailey** [comment]
I have done as you asked.

File: README.md [hunk 2]
```diff
@@ -11,152 +11,19 @@ An API wrapper for the Arionum cryptocurrency node.
...
 ## Change log
 
 Please see [CHANGELOG](CHANGELOG.md) for more information on what has changed recently.
 
 ## Testing
 
-```bash
-$ composer test
-```
+Please see [Testing Doc](documentation/testing.md) for more information on what has changed recently.
```
[the remark this answers is not in the document]
Comment (pxgamer) [reply README.md]: @LiamBailey, could you do this too? Then this seems fine to me 👍

**Comment by pxgamer** [comment]
Hi @LiamBailey, unfortunately I'm going to close this due to no update.

**Comment by LiamBailey** [comment]
@pxgamer According to my last comment I believed I had done all that you asked. If I missed something please do let me know. It wasn't my intention to bail on you in any way. Thanks

Liam

**Pull request closed without merging**